internal/proto/hotstuffpb/hotstuff.proto

```diff
@@ -81,6 +85,12 @@ message QuorumCert {
   QuorumSignature Sig = 1;
   uint64 View = 2;
   bytes Hash = 3;
+  LatencyVector LatencyVector = 4;
```

Comment:
Looks like you could do this instead, and avoid the indirections to get the map via the `LatencyVector` message type:
```proto
map<uint32, uint64> LatencyVector =4;
```
This will also avoid encoding the extra information related to the `LatencyVector` message type, such as its position number etc.

That said, I think we could represent this vector much more compactly.

1. Do you really need 64 bits for the latency (assuming ms granularity)?
  - With 32 bits you can represent `2^32/(1000*60*60*24) = 49` days.
  - With 24 bits you can represent `2^24/(1000*60*60) = 4.66` hours.
2. Similarly, I don't think we will ever approach 32 bit `hotstuff.ID`s in the vector / configuration.

Some alternative encodings:
```proto
repeated uint32 LatencyVector = 4;
repeated uint64 LatencyVector = 4;
bytes LatencyVector = 4;
```

With the first one, you could use lower 24 bits for the latency value, and the upper 8 bits for the `hotstuff.ID`. For the second one, I guess splitting it into two `uint32` values for each is fine. And the last one is even more flexible, but is perhaps a bit more awkward to work with.

To test out / support the first option, you would only need to do something like this in `hotstuffpb/convert.go`:
```go
func QuorumCertFromProto(qc *QuorumCert) hotstuff.QuorumCert {
	var hash hotstuff.Hash
	copy(hash[:], qc.GetHash())
	latencyVector := make(map[uint32]uint64)
	for _, val := range qc.GetLatencies() {
		id := val & 0xff00_0000 >> 24
		latency := val & 0x00ff_ffff
		latencyVector[id] = uint64(latency)
	}
	return hotstuff.NewQuorumCert(
		hotstuff.ID(qc.Creator),
		QuorumSignatureFromProto(qc.GetSig()),
		hotstuff.View(qc.GetView()),
		hash,
		latencyVector,
	)
```
And similar on the sending side.